src/EnergyPlus/VariableSpeedCoils.hh

```diff
@@ -284,6 +286,7 @@ namespace VariableSpeedCoils {
         // end variables for HPWH
         bool reportCoilFinalSizes; // one time report of sizes to coil selection report
         Real64 capModFacTotal;     // coil  TotCapTempModFac * TotCapAirFFModFac * TotCapWaterFFModFac, for result for simulation peak reporting
+        int AirLoopNum;
```

Comment:
This is uninitialized in the default ctor below. I would have set it to `0`, but I'm not sure if `0` has some sort of larger implications based on the discussion [here](https://github.com/NREL/EnergyPlus/pull/11243#discussion_r2426982148)